web/src/samples/complex/root/pages/index.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
+
+    <!-- Set the viewport width to match phone and tablet device widths -->
+    <meta name="viewport" content="width=device-width,user-scalable=no" />
+
+    <!-- Allow KeymanWeb to be saved to the iPhone home screen -->
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+
+    <!-- Enable IE9 Standards mode -->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+
+    <title>KeymanWeb Sample Page - Fully Compiled Source</title>
+
+    <!-- Your page CSS -->
+    <style type='text/css'>
+      body {font-family: Tahoma,helvetica;}
+      h3 {font-size: 1em;font-weight:normal;color: darkred; margin-bottom: 4px}
+      .test {font-size: 1.5em; width:80%; min-height:30px; border: 1px solid gray;}
+      #KeymanWebControl {width:50%;min-width:600px;}
+    </style>
+
+    <!-- KeymanWeb script -->
+    <script src="../js/keymanweb.js"></script>
+
+    <!--
+      For desktop browsers, a script for the user interface must be inserted here.
+
+      Standard UIs are toggle, button, float and toolbar.
+      The toolbar UI is best for any page designed to support keyboards for
+      a large number of languages.
+    -->
+    <script src="../js/kmwuitoggle.js"></script>
+
+    <!-- Initialization: set paths to keyboards, resources and fonts as required -->
+    <script>
+      var kmw=window.keyman;
+      kmw.init({
+        root: '..',
+        // Operates from the configured root set above - the 'root' folder for this sample.
+        resources: '/resources',
+        // If no absolute or clearly-relative specifiers are set within a keyboard's specified path,
+        // KMW will look for them here - two folders up from root, which is the base sample directory.
+        keyboards: '/../../',
+        attachType:'auto'
+      }).then(function() {
+        // Loads keyboards from the base sample folder
+        keyman.addKeyboards({
+          id:'us',
+          name:'English',
+          languages: {
+            id:'en',
+            name:'English'
+          },
+          // Note:  no './' or '../' prefix, which would root it based on the page's location, not the configured root above.
+          filename: 'us-1.0.js'
```

Comment:
After double-checking this again...

https://github.com/keymanapp/keyman/blob/18bb31bbc306717429545f71d7dcfe6eec909f53/web/src/app/web/kmwnative.ts#L78-L91

Looks like I'll need to patch up both this and the other scenario seen in the next `addKeyboards` block below.